src/lib.rs

```diff
@@ -97,6 +97,16 @@
 //!
 //! Check out the [custom_key](https://github.com/AaronErhardt/actix-governor/blob/main/examples/custom_key.rs) example to see how a custom key extractor can be implemented.
 //!
+//! # Customize response error
+//!
+//! By default, response error it's [`INTERNAL_SERVER_ERROR`] but if you want to change it you can override [`response_error`] function and return the error as you want
```

Comment:
My suggestion for improving the readability of the docs:

 //! By default, the response error generates an [`INTERNAL_SERVER_ERROR`] but if you want you can override the [`response_error`] function to return a custom error.